docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -22,16 +22,18 @@ A user can then install your collection on their systems.
 Creating a collection skeleton
 ==============================
 
-To start a new collection:
+To start a new collection, run the following command in your collections directory:
 
 .. code-block:: bash
 
-    collection_dir#> ansible-galaxy collection init my_namespace.my_collection
+    ansible_collections#> ansible-galaxy collection init my_namespace.my_collection
```

Comment:
I personally thought this change made it more clear that the collection my_namespace.my_collection is getting initialized in a directory that can contain multiple collections.

Reply:
okay. resolving.